onnxscript/function_libs/torch_aten/param_manipulation.py

```diff
@@ -0,0 +1,171 @@
+"""Function for manipulating input parameters of an Op or a OnnxFunction."""
+from __future__ import annotations
+
+import collections
+import dataclasses
+from typing import Any, List, OrderedDict, Sequence, Tuple
+
+import onnx
+
+from onnxscript import values
+
+# A special value to indicate that the default value is not specified
+_EmptyDefault = object()
+
+
+@dataclasses.dataclass(frozen=True)
+class ParamSchema:
+    """A schema for a parameter of an Op or a OnnxFunction.
+
+    Attributes:
+        name: The name of the parameter.
+        type: The type of the parameter.
```

Comment:
Since we have multiple type representations, which one is this supposed to be? Eg., is this supposed to be python-types (like "int" and "List[FLOAT]") ?